deployment/src/main/java/io/quarkiverse/logging/splunk/DevServicesLoggingSplunkProcessor.java

```diff
@@ -0,0 +1,188 @@
+            }
+            devService = null;
+            capturedDevServiceConfig = null;
+        }
+
+        // Re-initialize captured config and dev services.
+        capturedDevServiceConfig = config.devservices;
+
+        StartupLogCompressor compressor = new StartupLogCompressor(
+                (launchMode.isTest() ? "(test) " : "") + "Microcks Dev Services Starting:", consoleInstalledBuildItem,
+                loggingSetupBuildItem);
+        DevServicesResultBuildItem.RunningDevService newDevService;
+        try {
+            newDevService = startContainer(config.devservices, dockerStatusBuildItem,
+                    launchMode.getLaunchMode(), devServicesConfig.timeout);
+
+            if (devService == null) {
+                compressor.closeAndDumpCaptured();
+            } else {
+                compressor.close();
+                log.infof("The Splunk container is ready on %s", devService.getConfig().get(HANDLER_URL_CONFIG_PROP));
+            }
+        } catch (Throwable t) {
+            compressor.closeAndDumpCaptured();
+            throw new RuntimeException(t);
+        }
+
+        // Save started Dev Services.
+        devService = newDevService;
+
+        if (devService == null) {
+            return null;
+        }
+
+        if (first) {
+            first = false;
+            // Add close tasks on first run only.
+            Runnable closeTask = () -> {
+                if (devService != null) {
+                    try {
+                        devService.close();
+                    } catch (Throwable t) {
+                        log.error("Failed to stop microcks", t);
+                    }
+                }
+                first = true;
+                devService = null;
+                capturedDevServiceConfig = null;
+            };
+            closeBuildItem.addCloseTask(closeTask, true);
+        }
+
+        return devService.toBuildItem();
+    }
+
+    private DevServicesResultBuildItem.RunningDevService startContainer(DevServicesLoggingSplunkConfig config,
+            DockerStatusBuildItem dockerStatusBuildItem, LaunchMode launchMode, Optional<Duration> timeout) {
+        if (!isEnabled(config)) {
+            // explicitly disabled
+            log.info("Not starting devservices for Splunk as it has been disabled in the config");
+            return null;
+        }
+
+        if (!dockerStatusBuildItem.isDockerAvailable()) {
+            log.warn("Configure quarkus.log.handler.splunk.url or have a working docker daemon");
+            return null;
+        }
+
+        DockerImageName dockerImageName = DockerImageName.parse(config.imageName.orElse(SPLUNK_LATEST))
+                .asCompatibleSubstituteFor(SPLUNK_LATEST);
+
+        SplunkContainer splunkContainer = new SplunkContainer(dockerImageName, launchMode == DEVELOPMENT);
+
+        // Configure access to host - getting test-port from config or defaulting to 8081.
+        splunkContainer.withAccessToHost(true);
```

Comment:
do we really need access to the host ? 
we could find available port and map available ports in the container

Reply:
This part was complex coming from the microcks inspiration. Completely removed. And SplunkContainer simplified, hardcoding hostname to localhost with mapped port.